model/Service/Properties/serviceHostingCountries.md

```diff
@@ -9,10 +9,11 @@ Specifies a country code where a software service is hosted.
 ## Description
 
 This property should specify all countries where a service is located.  This includes indirect locations (e.g. where the data may be hosted).
-The string must be in the [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) format.
+The string must be in the [ISO 3166-1 alpha-3](https://www.iso.org/obp/ui/#iso:std:iso:3166:-3:ed-2:v1:en) format.  
```

Comment:
```suggestion
The string must be in the [ISO 3166-1 alpha-3](https://www.iso.org/obp/ui/#iso:std:iso:3166:-1) format.  
```

The current URL points to ISO 3166-3 which is "Part 3: Code for formerly used names of countries".

If a specific version of ISO 3166-1 is preferred, use https://www.iso.org/obp/ui/#iso:std:iso:3166:-1:ed-4:v1:en which is ISO 3166-1:2020 (latest).